src/modules/home/components/property/property-card.tsx

```diff
@@ -13,17 +15,48 @@ import 'swiper/css/pagination';
 import './swiper.css';
 
 import { EffectFade, Navigation, Pagination } from 'swiper/modules';
+import { axiosInstance } from '#/configs';
+import { useWishlistedProperties } from '../../api/get-wishlisted-properties';
 
-export function ApartmentCard(apartment: Accommodation) {
-  const handleFavoriteClick = (event: React.MouseEvent) => {
+export function ApartmentCard({ id, name, location, pricePerNight, images }: Accommodation) {
+  const { data } = useWishlistedProperties();
```

Comment:
I think this is inefficient because for every apartment card you are fetching all of the wishlisted properties again.